site/content/en/docs/enterprise/segment-anything-2-tracker.md

```diff
@@ -8,24 +8,54 @@ description: 'Accelerating video labeling using SAM2 model'
 ## Overview
 
 Segment Anything 2 is a segmentation model that allows fast and precise selection of any object in videos or images.
-For enterprise customers, this model can be installed in their self-hosted solution. To ensure a good experience,
-it is strongly recommended to deploy the model using a GPU. Although it is possible to use a CPU-based version,
-it generally performs much slower and is suitable only for handling a single parallel request. Unlike a regular
-tracking model, the SAM 2 tracker is implemented as an annotation action. This allows it to be applied to existing
-objects (polygons and masks) to track them forward for a specified number of frames.
+SAM2 tracking is available in two implementations:
+
+1. **Classic SAM2 Tracker (Annotation Action)**: Available for Enterprise self-hosted Basic/Premium deployments.
+This is implemented as an annotation action using serverless functions and requires Nuclio deployment.
+
+1. **AI Agent SAM2 Tracker**: Available for CVAT Online and Enterprise 2.42.0+ via auto-annotation (AA) functions
```

Comment:
Shall we specify enterprise version here? The all documentation is anyway bound to the specific version.